Index: org/w3c/css/css/StyleSheetGenerator.java
===================================================================
RCS file: /sources/public/2002/css-validator/org/w3c/css/css/StyleSheetGenerator.java,v
retrieving revision 1.17
diff -u -d -p -r1.17 StyleSheetGenerator.java
--- org/w3c/css/css/StyleSheetGenerator.java	9 Apr 2008 13:02:52 -0000	1.17
+++ org/w3c/css/css/StyleSheetGenerator.java	1 Jul 2008 22:00:26 -0000
@@ -57,7 +57,7 @@ public class StyleSheetGenerator extends
 	URL url;
 	availableFormat = new Utf8Properties();
 	try {
-	    url = Messages.adjustURL(StyleSheetGenerator.class.getResource("format.properties"));
+	    url = StyleSheetGenerator.class.getResource("format.properties");
 	    java.io.InputStream f = url.openStream();
 	    availableFormat.load(f);
 	    f.close();
@@ -68,7 +68,7 @@ public class StyleSheetGenerator extends
 
 	availablePropertiesURL = new Utf8Properties();
 	try {
-	    url = Messages.adjustURL(StyleSheetGenerator.class.getResource("urls.properties"));
+	    url = StyleSheetGenerator.class.getResource("urls.properties");
 	    java.io.InputStream f = url.openStream();
 	    availablePropertiesURL.load(f);
 	    f.close();
@@ -158,6 +158,7 @@ public class StyleSheetGenerator extends
 	produceStyleSheet();
 
 	try {
+	    Velocity.init();
 	    template = Velocity.getTemplate(template_file);
 	} catch (ResourceNotFoundException rnfe) {
 	    System.err.println(rnfe.getMessage());
Index: org/w3c/css/util/Messages.java
===================================================================
RCS file: /sources/public/2002/css-validator/org/w3c/css/util/Messages.java,v
retrieving revision 1.19
diff -u -d -p -r1.19 Messages.java
--- org/w3c/css/util/Messages.java	1 Jul 2008 13:43:04 -0000	1.19
+++ org/w3c/css/util/Messages.java	1 Jul 2008 22:00:26 -0000
@@ -163,7 +163,7 @@ public class Messages {
 	static {
 		Utf8Properties tmp;
 		try {
-			URL url = adjustURL(Messages.class.getResource("Messages.properties.de"));
+			URL url = Messages.class.getResource("Messages.properties.de");
 			java.io.InputStream f = url.openStream();
 			try {
 				tmp = new Utf8Properties();
@@ -184,7 +184,7 @@ public class Messages {
 		// ------------------------------------------------
 
 		try {
-			URL url = adjustURL(Messages.class.getResource("Messages.properties.en"));
+			URL url = Messages.class.getResource("Messages.properties.en");
 			java.io.InputStream f = url.openStream();
 			try {
 				tmp = new Utf8Properties();
@@ -202,7 +202,7 @@ public class Messages {
 		// ------------------------------------------------
 
 		try {
-			URL url = adjustURL(Messages.class.getResource("Messages.properties.es"));
+			URL url = Messages.class.getResource("Messages.properties.es");
 			java.io.InputStream f = url.openStream();
 			try {
 				tmp = new Utf8Properties();
@@ -221,7 +221,7 @@ public class Messages {
 		// -----------------------
 
 		try {
-			URL url = adjustURL(Messages.class.getResource("Messages.properties.fr"));
+			URL url = Messages.class.getResource("Messages.properties.fr");
 			java.io.InputStream f = url.openStream();
 			try {
 				tmp = new Utf8Properties();
@@ -240,7 +240,7 @@ public class Messages {
 		// -----------------------
 
 		try {
-			URL url = adjustURL(Messages.class.getResource("Messages.properties.ko"));
+			URL url = Messages.class.getResource("Messages.properties.ko");
 			java.io.InputStream f = url.openStream();
 			try {
 				tmp = new Utf8Properties();
@@ -258,7 +258,7 @@ public class Messages {
 		// -----------------------
 		
 		try {
-			URL url = adjustURL(Messages.class.getResource("Messages.properties.it"));
+			URL url = Messages.class.getResource("Messages.properties.it");
 			java.io.InputStream f = url.openStream();
 			try {
 				tmp = new Utf8Properties();
@@ -276,7 +276,7 @@ public class Messages {
 		// -----------------------
 
 		try {
-			URL url = adjustURL(Messages.class.getResource("Messages.properties.nl"));
+			URL url = Messages.class.getResource("Messages.properties.nl");
 			java.io.InputStream f = url.openStream();
 			try {
 				tmp = new Utf8Properties();
@@ -294,7 +294,7 @@ public class Messages {
 		// -----------------------
 
 		try {
-			URL url = adjustURL(Messages.class.getResource("Messages.properties.ja"));
+			URL url = Messages.class.getResource("Messages.properties.ja");
 			java.io.InputStream f = url.openStream();
 			try {
 				tmp = new Utf8Properties();
@@ -312,7 +312,7 @@ public class Messages {
 		// -----------------------
 		
 		try {
-			URL url = adjustURL(Messages.class.getResource("Messages.properties.pl-PL"));
+			URL url = Messages.class.getResource("Messages.properties.pl-PL");
 			java.io.InputStream f = url.openStream();
 			try {
 				tmp = new Utf8Properties();
@@ -332,7 +332,7 @@ public class Messages {
 		// -----------------------
 		
 		try {
-			URL url = adjustURL(Messages.class.getResource("Messages.properties.pt-BR"));
+			URL url = Messages.class.getResource("Messages.properties.pt-BR");
 			java.io.InputStream f = url.openStream();
 			try {
 				tmp = new Utf8Properties();
@@ -353,7 +353,7 @@ public class Messages {
 		// -----------------------
 
 		try {
-			URL url = adjustURL(Messages.class.getResource("Messages.properties.zh-cn"));
+			URL url = Messages.class.getResource("Messages.properties.zh-cn");
 			java.io.InputStream f = url.openStream();
 			try {
 				tmp = new Utf8Properties();
@@ -371,21 +371,4 @@ public class Messages {
 		}
 
 	}
-
-	/**
-	 * This method is called to clean the URL
-	 * /path/to/file will , file://localhost/path/to/file, file://C:\path\to\file, ...
-	 * will become file:///path/to/file 
-	 * @param resource, the URL to "clean"
-	 * @return the clean URL
-	 * @throws MalformedURLException
-	 */
-	public static URL adjustURL(URL resource) throws MalformedURLException {
-		String urlStr = resource.getFile();
-		if (urlStr.startsWith("file://"))
-			urlStr.replaceFirst("file://localhost", "file://");
-		else
-			urlStr = "file:///" + urlStr;
-		return new URL(urlStr);
-	}
 }
Index: org/w3c/css/util/xml/XMLCatalog.java
===================================================================
RCS file: /sources/public/2002/css-validator/org/w3c/css/util/xml/XMLCatalog.java,v
retrieving revision 1.5
diff -u -d -p -r1.5 XMLCatalog.java
--- org/w3c/css/util/xml/XMLCatalog.java	13 Jul 2007 13:32:21 -0000	1.5
+++ org/w3c/css/util/xml/XMLCatalog.java	1 Jul 2008 22:00:26 -0000
@@ -36,7 +36,7 @@ public class XMLCatalog extends Utf8Prop
 	public XMLCatalog() {
 		try {
 			baseURI = XMLCatalog.class.getResource("catalog.xml");
-			parse("\"" + baseURI.toString() + "\"");
+			parse(baseURI.toString());
 		} catch (Exception e) {
 			System.err.println("org.w3c.css.css.XMLStyleSheetHandler: "
 					+ "couldn't load catalog.xml");
